api/tests/test_api.py

```diff
@@ -234,3 +235,53 @@ async def test_push_points():
             assert response.status_code == 200
             assert mp_unpackb(response._content) == "data sent"
         del ps
+
+
+class TrialA(BaseModel):
+    integers: list[int]
+    floats: list[float]
+
+
+class TrialB(BaseModel):
+    integers: list[int]
+    floats: list[float]
+    original: TrialA
+
+
+@app.post("/test_pydantic")
+@message_unpack
+async def test_pydantic(data: TrialA) -> TrialB:
```

Comment:
Does this need marking with `@pytest.mark.asyncio`

Reply:
No, it's a test endpoint and gets renamed in later PR.